Monika After Story/game/script-story-events.rpy

```diff
@@ -656,9 +658,10 @@ label mas_random_ask:
             m 1eua "Great!"
             m "If you get tired of watching me talk about the same things over and over,{w} just open up the settings and uncheck 'Repeat Topics'."
             # TODO: this really should be a smug or wink face
```

Comment:
To be perfectly honest, I don't agree with this todo. That seems more like something she'd be *concerned* about, not smug about.